packages/react-components/eslint-plugin-react-components/tsconfig.lib.json

```
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false,
+    "lib": ["ES2019"],
+    "declaration": true,
+    "declarationDir": "../../../dist/out-tsc/types",
+    "outDir": "../../../dist/out-tsc",
+    "inlineSources": true,
+    "types": ["static-assets", "environment", "node"],
```

we don't wanna leverage this , no static assets should be allowed to be loaded within the plugin

```suggestion
    "types": ["environment", "node"],
```

---

removed